### PR TITLE
Setup bookstore project

### PR DESCRIPTION
**Pull request by omar-muhamad**
- [ ] Create react app & install react-redux & remove boilerplate files
- [ ] Remove all unused code.
- [ ] Add basic project components & redux files
- [ ] Edit readme and fix linters

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  "dependencies": {
+    "@testing-library/jest-dom": "^5.16.1",
+    "@testing-library/react": "^12.1.2",
+    "@testing-library/user-event": "^13.5.0",
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2",
+    "react-router-dom": "^6.2.1",
+    "react-scripts": "5.0.0",
+    "web-vitals": "^2.1.3"
+  },
```
Comment (elyor-doniyorov) [inline package.json]: - [ ] Please install `react-redux` dependency by running `npm install react-redux`. `react-redux` will be needed to use Redux in the next milestones

File: src/components/Books.js [hunk 2]
```diff
@@ -0,0 +1,3 @@
+const Books = () => <h2>Books List Under construction</h2>;
```
Comment (elyor-doniyorov) [inline src/components/Books.js]: - [ ] Books should have a form for adding a book (no functionality yet). Let's create a basic form and display it.

File: README.md [hunk 3]
```diff
@@ -1 +1,55 @@
...
+
+- `Npm run test`
+  
+## Deployment
+
+1. Clone the project using git-bash or Githup Desktop.
+2. Open the project folder with VSCode or any Editor.
+3. Open terminal and navigate to the project folder.
+4. Type `npm install`.
+5. Type `npm start`.
```
Comment (elyor-doniyorov) [inline README.md]: - [ ] I think you can change the section title from `Deployment `to `Setup ` to make it meaningful.

**Review by elyor-doniyorov — CHANGES_REQUESTED** [review]
Hi @Omar-Muhamad, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
almost-there


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by BrianSammit — APPROVED** [review]
# STATUS: APROVED :green_circle: :clap: 

Hello @Omar-Muhamad, 
Your project is complete!  There is nothing else to say other than... Great job 👏 
Congratulations! 🎉 

Now you can merge this branch and move on :1st_place_medal: 
HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers: :four_leaf_clover: 

well-1

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**